daprdocs/content/en/reference/components-reference/supported-bindings/graghql.md

```diff
@@ -0,0 +1,72 @@
+---
+type: docs
+title: "GraphQL binding spec"
+linkTitle: "GraphQL"
+description: "Detailed documentation on the GraphQL binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/graphql/"
+---
+
+## Component format
+
+To setup GraphQL binding create a component of type `bindings.graphql`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: example.bindings.graphql
+spec:
+  type: bindings.graphql
+  version: v1
+  metadata:
+    - name: endpoint
+      value:  #required    
+    - name: header:xyz # e.g. header:x-hasura-access-key
+      value:
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Binding support |  Details | Example |
+|--------------------|:--------:|------------|-----|---------|
+| endpoint | Y | Output | GraphQL endpoint string See [here](#url-format) for more details | `"http://localhost:4000/graphql"` |
+| header | N | Output | GraphQL header | `"header:Cache-Control"` |
```

Comment:
Here it looks like value is `"header:Cache-Control"`, whereas above you have it as the name. Please update with correct syntax.